src/dev.html

```diff
@@ -18,6 +18,8 @@
 
     <header id="site-header"></header>
 
+    <mkt-nav role="navigation" id="site-nav"></mkt-nav>
```

Comment:
This should be rendered via JS in the site-header. When this is deployed, mkt/commonplce/templates/commonplace/index.html will be served, which won't have this.